src/app_charts/base/cloud/apps-crd.yaml

```diff
@@ -36,7 +36,7 @@ spec:
                           type: string
                         inline:
                           type: string
-                    robots:
+                    robot:
```

Comment:
In line 94 below it is also robots:

Reply:
I think robots (plural) is correct and also what we actually use everywhere

Reply:
Prometheus for example calls it [prometheus-robot](https://github.com/googlecloudrobotics/core/blob/master/src/app_charts/prometheus/BUILD.bazel#L40), if I understand the underlying bazel function correctly.
And the [go type](https://github.com/googlecloudrobotics/core/blob/master/src/go/pkg/apis/apps/v1alpha1/types.go#L129) calls it robot too. That's why we call it robot in all our app charts too.

Reply:
We use `robot` in [App CRs](https://github.com/SAP/ewm-cloud-robotics/blob/master/applayer-yaml/ewm-robot-controller/app.yaml#L13) since the beginning and it would be more effort to change it here.
In the end there was no other way to use the name of the Go structure and we just did not notice the inconsistency with App CRD specification from app-crd.yaml because CRDs in v1beta1 preserved unknown fields.

Reply:
RIght, this is for the app! Makes sense.